### PR TITLE
Hw 5

### PR DESCRIPTION
**Pull request by AntonBakaiev**

### CODE REVIEW

File: students/AntonBakaiev/hw-4/answer.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+let initialText = prompt('Введите урывок текста');
```
Comment (Temu4) [inline students/AntonBakaiev/hw-4/answer.js:1]: Please remove files related with previous HW

File: students/AntonBakaiev/hw-5/answer.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+Task 1
+
+function getMaxNumber(...number) { 
+    let max = Math.max(...number)
```
Comment (Temu4) [inline students/AntonBakaiev/hw-5/answer.js:4]: Why not `const`?

File: students/AntonBakaiev/hw-5/answer.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    return max
+}
+
+console.log(getMaxNumber(2, 0)); //2
+console.log(getMaxNumber(2, 0, 5)); // 5
+console.log(getMaxNumber(2, 0, 5,-1,3,10,4)); // 10
+
+
+
+Task 2
```
Comment (Temu4) [inline students/AntonBakaiev/hw-5/answer.js:14]:
```suggestion
// Task 2 
```

File: students/AntonBakaiev/hw-5/answer.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+console.log(getMaxNumber(2, 0, 5,-1,3,10,4)); // 10
+
+
+
+Task 2 
+
+function joinObjects(obj1, obj2, ...obj) { 
+    let objSum = Object.assign(obj1, obj2, ...obj)
+    return objSum
+}
```
Comment (Temu4) [inline students/AntonBakaiev/hw-5/answer.js:19]:
```suggestion
function joinObjects(...objects) { 
    const objSum = Object.assign(...objects);
    return objSum;
}
```

File: students/AntonBakaiev/hw-5/answer.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+console.log(joinObjects({ name: 'Nike' }, { title: 'boss' })); 
+console.log(joinObjects({ name: 'Nike' }, { title: 'boss' }, { age: 30})); 
+console.log(joinObjects({ name: 'Nike' }, { title: 'boss' }, { age: 30 }, { position: 'CEO' }, { isDefault: false })); 
+
+
+Task 3
```
Comment (Temu4) [inline students/AntonBakaiev/hw-5/answer.js:26]:
```suggestion
// Task 3
```

File: students/AntonBakaiev/hw-5/answer.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+const student = {
+    name: 'Anna',
+    age: 20
+};
+
+Object.defineProperties(student, 
+    { 'toString': {
+    enumerable: false,
```
Comment (Temu4) [inline students/AntonBakaiev/hw-5/answer.js:35]: Using `Object.defineProperties` `enumerable` would be `false` by default.

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: students/AntonBakaiev/hw-5/answer.js (hunk 6 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: виправив

File: students/AntonBakaiev/hw-5/answer.js (hunk 5 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: +

File: students/AntonBakaiev/hw-5/answer.js (hunk 4 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: +

File: students/AntonBakaiev/hw-5/answer.js (hunk 3 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: +

File: students/AntonBakaiev/hw-5/answer.js (hunk 2 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: виправив

File: students/AntonBakaiev/hw-5/answer.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
+Task 1
```
[the remark this answers is not in the document]
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-5/answer.js]: +

**Review by Temu4 — APPROVED** [review]

**Pull request merged**